src/libs/antares/study/parts/hydro/series.cpp

```diff
@@ -236,11 +236,22 @@ void DataSeriesHydro::reset()
     count = 1;
 }
 
-void DataSeriesHydro::resizeRORandSTORAGE(unsigned int width)
+void DataSeriesHydro::resize_ROR_STORAGE_MINGEN_whenGeneratedTS(unsigned int newWidth)
 {
-    ror.resize(width, HOURS_PER_YEAR);
-    storage.resize(width, DAYS_PER_YEAR);
-    count = width;
+    ror.resize(newWidth, HOURS_PER_YEAR);
+    storage.resize(newWidth, DAYS_PER_YEAR);
+
+    // Resizing mingen : new extra columns are copies of the first column.
+    //    Note that mingen has necessarily one column at least, by construction
```

Comment:
Why do we want to do that ?

Reply:
Please see [this comment](https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/1752#discussion_r1417403948)

Reply:
> Why do we want to do that ?

Please see [this comment](https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/1752#discussion_r1417403948)